lib/liquid/partial_cache.rb

```diff
@@ -2,18 +2,22 @@
 
 module Liquid
   class PartialCache
-    def self.load(template_name, context:, parse_context:)
+    def self.load(template_name, context:, parse_context:, caller:)
```

Comment:
This shadows the `caller` built-in; maybe a clearer and more unique name would be `caller_tag_name`.

Reply:
Thanks for looking into this @pushrax but we decided to take another approach that doesn't require changes here in `liquid`